recipes/pannenkoek/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: pannenkoek
+  version: "0.1.6"
+
+source:
+  fn: pannenkoek-0.1.6.tar.gz
+  url: https://pypi.python.org/packages/f5/7b/fea1766fcdc91629e0afcb503b05262ce8b30088c8e9ebf867b962f32767/pannenkoek-0.1.6.tar.gz
+  md5: f712dbc03ed28d3109d56df231caeca5
+build:
+  # number: 1
+  skip: True  # [not py27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - qiime
+    - rpy2
+    - numpy
+    - matplotlib
+    - argparse
```

Comment:
Is argparse really needed? I guess it is annotated in the setup.py, but unless we do want to support python2.6 this should be removed from here and the setup.py